src/explorer/LogStreamTreeItem.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { getLogStreamId, getLogStreams, ILogStream, ISiteTreeRoot } from 'vscode-azureappservice';
+import { AzureParentTreeItem, AzureTreeItem } from 'vscode-azureextensionui';
+
+const openLogStreamCommand: string = 'appService.OpenLogStream';
+const stopLogStreamCommand: string = 'appService.StopLogStream';
+const disconnectLogStream: string = 'Disconnect log stream...';
```

Comment:
I was talking to Matt and he thought it would be more useful in the "isConnected" case if this node opened the output channel. Sometimes it's hard to find.

Honestly "Connect to log stream..." and "View log stream..." are so similar you might not even need to change the label if that simplifies the code